Assets/Scripts/Possession/PossessionController.cs

```diff
@@ -51,15 +51,21 @@ public void Unpossess()
     {
         if (CurrentPossession != null)
         {
-            CurrentPossession.GetComponent<IPossessable>().Unpossess();
             _thirdPersonController.toUnpossessLocation();
-            CurrentPossession = null;
-            CurrentThrowable = null;
+            RemovePossession();
             _thirdPersonController.freeze = false;
             CurrentPossessionChanged?.Invoke();
         }
     }
 
+    private void RemovePossession()
```

Comment:
the name of the function makes it sound like it does the same thing as Unpossess()